src/components/common/TmBalance.vue

```diff
@@ -457,6 +479,55 @@ export default {
 }
 </script>
 <style scoped>
+select {
+  background: var(--input-bg);
+  color: var(--txt, #333);
+  border: none;
+}
+
+select option {
+  background: var(--app-bg);
+  color: var(--txt);
+  font-family: var(--sans);
+}
+
+.currency-flag {
+  width: 15px;
+  margin-right: 3px;
```

Comment:
`0.25rem` (in lunie) is `4px` which is probably fine!

```suggestion
  margin-right: 0.25rem;
```